rest-api-spec/src/main/resources/rest-api-spec/api/_global.json

```diff
@@ -0,0 +1,29 @@
+{
+  "description": "Parameters that are accepted by all API endpoints.",
+  "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html",
+  "params": {
+    "pretty": {
+      "type": "boolean",
+      "description": "Pretty format the returned JSON response.",
+      "default": false
+    },
+    "human": {
+      "type": "boolean",
+      "description": "Return human readable values for statistics.",
+      "default": true
+    },
+    "error_trace": {
+      "type": "boolean",
+      "description": "Include the stack trace of returned errors.",
+      "default": false
+    },
+    "source": {
+      "type": "string",
+      "description": "The URL-encoded request definition. Useful for libraries that do not accept a request body for non-POST requests."
+    },
+    "filter_path": {
```

Comment:
@javanna @clintongormley I'm not sure it belongs here the more I think about it.  It's not really part of the spec, in that Elasticsearch doesn't support it.  If I'm understanding the use of `ignore`, then it's really a mechanism of the yaml tests and not the Elasticsearch REST API.  Moreover, the nice thing about adding this common file to the spec is that it removes the burden of maintaining a list global parameters from the client.  Right now the .NET client maintains a whitelist internally so that it can automatically generate methods, but if we start adding unsupported parameters like `ignore`, then we'll have maintain a blacklist instead to prevent from generating unsupported methods - and we're back to square one.